### PR TITLE
reverse discount design, option to hide discount

### PR DESCRIPTION
**Pull request by vsrisuknimit**
image

image

### CODE REVIEW

**Comment by vsrisuknimit** [comment]
Keeping the second price up high. Blocking card name is better than blocking card text or graphics. I like grayscale better than transparency.

**Review by sienmich — APPROVED** [review]
Thanks!

**Comment by sienmich** [comment]
Is there some default value for this option? I think they can be turned on by default.

File: src/components/Preferences.ts [hunk 1]
```diff
@@ -74,6 +74,7 @@ export const Preferences = Vue.component('preferences', {
       'smooth_scrolling': false as boolean | unknown[],
       'hide_tile_confirmation': false as boolean | unknown[],
       'show_card_number': false as boolean | unknown[],
+      'show_discount_on_cards': true as boolean | unknown[],
```
Comment (vsrisuknimit) [inline src/components/Preferences.ts:77]: @sienmich The discount is on by default.

Comment (sienmich) [reply src/components/Preferences.ts:77]: Thanks, I was looking for this one

**Pull request merged**